### PR TITLE
fix : after review

### PR DESCRIPTION
**Pull request by al4iii**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,122 +1,113 @@
...
+const clearBtns = document.querySelectorAll('.clear-btn');   
+const resultBtn = document.getElementById('result');
+const piBtn = document.getElementById('pi');
+const display =document.getElementById('display');
+let memoryCurrentNumber = 0; // what number is currently entered on the scoreboard, by default 0
+let memoryNewNumber = false; // have we introduced a new meaning?
+let memoryPendingOperation= '';    // pending operation
     
-for(let i=0; i<numbers.length; i++){
+for (let i=0; i<numbers.length; i++) {
```
Comment (Yauhenia-Zhdanovich) [inline script.js:12]: `for (let i = 0; i < numbers.length; i++) {`

File: script.js [hunk 2]
```diff
@@ -1,122 +1,113 @@
...
   let number = numbers[i];
-  number.addEventListener('click', function(e){ //event handler
+  number.addEventListener('click', function(e) { //event handler
     numberPress(e.target.textContent)
```
Comment (Yauhenia-Zhdanovich) [inline script.js:15]: ;

File: script.js [hunk 3]
```diff
@@ -1,122 +1,113 @@
...
+
+function numberPress(number) {
+  if (memoryNewNumber) {
     display.value = number;
-  }else{
-    display.value += number;
+    memoryNewNumber = false;
+  } else {
+  display.value === '0' ? display.value = number : display.value = display.value + number;
   };
```
Comment (Yauhenia-Zhdanovich) [inline script.js:43]: ` display.value = display.value === '0' ? number : display.value + number;`

File: script.js [hunk 4]
```diff
@@ -1,122 +1,113 @@
...
+const numbers = document.querySelectorAll('.number');
+const operations = document.querySelectorAll('.operator');
+const decimalBtn = document.getElementById('decimal');
+const clearBtns = document.querySelectorAll('.clear-btn');   
+const resultBtn = document.getElementById('result');
+const piBtn = document.getElementById('pi');
+const display =document.getElementById('display');
+let memoryCurrentNumber = 0; // what number is currently entered on the scoreboard, by default 0
+let memoryNewNumber = false; // have we introduced a new meaning?
+let memoryPendingOperation= '';    // pending operation
```
Comment (Yauhenia-Zhdanovich) [inline script.js:10]: watch out for spaces, pls

File: script.js (hunk 2 above)
Comment (al4iii) [reply script.js:15]: done

File: script.js (hunk 3 above)
Comment (al4iii) [reply script.js:43]: done

File: script.js (hunk 4 above)
Comment (al4iii) [reply script.js:10]: done

File: script.js (hunk 1 above)
Comment (al4iii) [reply script.js]: done

**Pull request merged**